presto-spi/src/main/java/com/facebook/presto/spi/relation/ConstantExpression.java

```diff
@@ -35,9 +35,7 @@
     private final Object value;
     private final Type type;
 
-    public ConstantExpression(
-            Optional<SourceLocation> sourceLocation,
-            Object value, Type type)
+    public ConstantExpression(Optional<SourceLocation> sourceLocation, Object value, Type type)
```

Comment:
I would leave this as is and maybe add a line break after `Object value,` for readability instead of keeping everything on the same line.

Reply:
Single lines for few parameters is better.